app/webpacker/lib/requests/routes.js.erb

```diff
@@ -259,7 +255,11 @@ export const panelUrls = {
   },
 }
 
-export const panelPageUrl = (panelPage) => `<%= CGI.unescape(Rails.application.routes.url_helpers.panel_page_path(id: "${panelPage}")) %>`;
+export const panelPageUrl = (panelPage, queryParams = {}) => {
+  const queryString = jsonToQueryString(queryParams);
+  const urlSuffix = queryString ? `?${queryString}` : '';
```

Comment:
Is this necessary? I think `www.test.com/something/some_page/panel?` is a perfectly valid URL and Rails can parse it.

Reply:
Oh ya you are right. I've changed it.